deployments/common/image/default-home-contents/.astropy/config/astroquery.cfg

```diff
@@ -1,5 +1,216 @@
+#sort_output_by = RA or Longitude
+
+[nist]
+
+# Name of the NIST URL to query.
+# Options: http://physics.nist.gov/cgi-bin/ASD/lines1.pl
+#nist_server = http://physics.nist.gov/cgi-bin/ASD/lines1.pl
+
+# time limit for connecting to NIST server
+#timeout = 30
+
+[nrao]
+
+# Name of the NRAO mirror to use.
+# Options: https://archive.nrao.edu/archive/ArchiveQuery
+#nrao_server = https://archive.nrao.edu/archive/ArchiveQuery
+
+# time limit for connecting to NRAO server
+#timeout = 60
+
+[nvas]
+
+# Name of the NVAS mirror to use.
+# Options: https://webtest.aoc.nrao.edu/cgi-bin/lsjouwer/archive-pos.pl
+#nvas_server = https://webtest.aoc.nrao.edu/cgi-bin/lsjouwer/archive-pos.pl
+
+# time limit for connecting to NVAS server
+#timeout = 60
+
+[ogle]
+
+# Name of the OGLE mirror to use.
+# Options: http://ogle.astrouw.edu.pl/cgi-ogle/getext.py
+#ogle_server = http://ogle.astrouw.edu.pl/cgi-ogle/getext.py
+
+# Time limit for connecting to the OGLE server.
+#timeout = 60
+
+[sdss]
+
+# Link to SDSS website.
+#sdss_server = http://data.sdss3.org/sas/dr10
+
+# Max number of queries allowed per second
+#maxqueries = 1
+
+# Default timeout for connecting to server
+#timeout = 30
+
+[simbad]
+
+# Name of the SIMBAD mirror to use.
+# Options: simbad.u-strasbg.fr, simbad.harvard.edu
+#simbad_server = simbad.u-strasbg.fr
+
+# time limit for connecting to Simbad server
+#timeout = 60
+
+# maximum number of rows that will be fetched from the result.
+#row_limit = 0
+
+[splatalogue]
+
+#Splatalogue SLAP interface URL (not used). = http://find.nrao.edu/splata-slap/slap
+
+#Splatalogue web interface URL. = http://www.cv.nrao.edu/php/splat/c_export.php
+
+# default timeout for connecting to server
+#timeout = 60
+
+#Limit to number of lines exported = 10000
+
+[template_module]
+
+# put a brief description of the item here
+# Options: http://dummy_server_mirror_1, http://dummy_server_mirror_2, http://dummy_server_mirror_n
+#server = http://dummy_server_mirror_1
+
+# default timeout for connecting to server
+#timeout = 30
+
+[ukidss]
+
+# Name of the UKIDSS mirror to use.
+# Options: http://surveys.roe.ac.uk:8080/wsa/
+#ukidss_server = http://surveys.roe.ac.uk:8080/wsa/
+
+# time limit for connecting to UKIDSS server
+#timeout = 60
+
+[vizier]
+
+# Name of the VizieR mirror to use.
+# Options: vizier.u-strasbg.fr, vizier.nao.ac.jp, vizier.hia.nrc.ca, vizier.ast.cam.ac.uk, vizier.cfa.harvard.edu, www.ukirt.jach.hawaii.edu, vizier.iucaa.ernet.in, vizier.china-vo.org
+#vizier_server = vizier.u-strasbg.fr
+
+# default timeout for connecting to server
+#timeout = 60
+
+# maximum number of rows that will be fetched from the result (set to -1 for
+# unlimited).
+#row_limit = 50
+
+[astrometry_net]
+
+# The Astrometry.net API key
+api_key =
+
 [mast]
 
 # Override MAST server with JWST A-string so we have access to
 # relevant data:
-server = https://pwjwdmsauiweb.stsci.edu
+# server = https://pwjwdmsauiweb.stsci.edu
```

Comment:
Why was this done?

Reply:
Good question.   While updating astroquery.cfg,   I figured it was a 1.5 year old customization which was probably no longer valid.   But I should have researched more.   Fine to revert it,  your call.

Reply:
Please leave that server override commented out -- it breaks astroquery functionality in all the envs.  We may need to do more with custom MAST servers down the road with Roman, but for now this customization should be omitted.